### PR TITLE
Created Boilerplates tutorial

### PR DESCRIPTION
**Pull request by evildmp**

### CODE REVIEW

File: introduction/08-create-custom-boilerplate.rst [hunk 1]
```diff
@@ -0,0 +1,512 @@
+.. _tutorial-create-boilerplate:
+
+Create a custom Boilerplate
+===========================
+
+Each Divio Cloud site has a frontend *Boilerplate*, a set of default templates
+and static file.
```
Comment (vxsx) [inline introduction/08-create-custom-boilerplate.rst]: files?

File: introduction/08-create-custom-boilerplate.rst [hunk 2]
```diff
@@ -0,0 +1,512 @@
...
+
+
+Add an *application-specific* template (``polls/base.html``)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The polls application knows nothing of the new templates we have created.
+
+If you look at the polls application, you will see that each of its view
+templates (the index view, the detail view and so on) extend its own, minimal
+``polls/templates/pollls/base.html`` file, which contains nothing but:
```
Comment (vxsx) [inline introduction/08-create-custom-boilerplate.rst]: typo pollls

File: introduction/08-create-custom-boilerplate.rst [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+  be unique on the Divio Cloud system.
+* ``templates`` are only required for Boilerplates intended to be used with
+  django CMS.
+* ``identifier`` is a namespace, that will allow applications that are
+  Boilerplate-aware to build in support for particular Boilerplates into their
+  own frontend code. An example of this is `Aldryn News & Blog
+  <https://github.com/aldryn/aldryn-newsblog/tree/master/aldryn_newsblog>`_.
+* The ``version`` is for your own convenience.
+
+Run the ``boilerplate verify`` command to check that the ``boilerplate.json``
```
Comment (vxsx) [inline introduction/08-create-custom-boilerplate.rst]: i'm personally a bit confused about `boilerplate verify`, it's not a real command but is marked like one

File: introduction/08-create-custom-boilerplate.rst [hunk 4]
```diff
@@ -0,0 +1,512 @@
+.. _tutorial-create-boilerplate:
+
+Create a custom Boilerplate
+===========================
+
+Each Divio Cloud site has a frontend *Boilerplate*, a set of default templates
+and static file.
+
+Typically, a Boilerplate will define how the Django templates are structured and
+make opinionated choices about what JavaScript frameworks and CSS tools are
```
Comment (FinalAngel) [inline introduction/08-create-custom-boilerplate.rst]: "What CSS and JavaScript frameworks are used."

File: introduction/08-create-custom-boilerplate.rst (hunk 3 above)
Comment (FinalAngel) [reply introduction/08-create-custom-boilerplate.rst]: Is it verify or validate? underneath is validate

File: introduction/08-create-custom-boilerplate.rst [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+
+This is required before your Boilerplate can be uploaded.
+
+This is a `2-Clause BSD "Simplified" License
+<https://opensource.org/licenses/BSD-2-Clause>`_.
```
Comment (FinalAngel) [inline introduction/08-create-custom-boilerplate.rst]: maybe add info on what other license you might choose?

File: introduction/08-create-custom-boilerplate.rst [hunk 6]
```diff
@@ -0,0 +1,512 @@
...
+    It would be wise to turn your ``tutorial-boilerplate`` directory into a Git
+    repository, so you can track changes in it. (This is what we do with
+    :ref:`the provided Divio Cloud Boilerplates <about-boilerplates>`).
+
+
+When you make changes to your Boilerplate, increment its ``version`` in the
+``boilerplate.json`` and upload it to the Control Panel by running the
+``boilerplate upload`` command on the new version.
+
+You'll find its *Versions* listed on its page in the Control Panel.
```
Comment (FinalAngel) [inline introduction/08-create-custom-boilerplate.rst]: Add the info that boilerplates do not update like addons, it's a one-way-street

**Comment by evildmp** [comment]
@vxsx http://divio-cloud-developer-handbook.readthedocs.io/en/latest/background/about-boilerplates.html#how-boilerplates-work has some explanation of more complex matters, but I'll also later be adding a how-to guide that covers that too.

**Pull request merged**